### ISSUE TITLE
Can't find GetTargetFrameworks

### ISSUE THREAD

**Issue by JoshClose** [nuke-build/nuke#324]
I'm following the example for [How can I test/publish for multiple configurations/frameworks/runtimes?](https://nuke.build/docs/getting-started/faq.html#how-can-i-testpublish-for-multiple-configurationsframeworksruntimes) and `GetMSBuildProject()` doesn't seem to have the method `GetTargetFrameworks()` on it. I can't seem to find any information about this from googling either. Searching this repository gave me 0 results too.

My `project` is got from `Solution.GetProject("MyProjectName")`.

I set up the build project using `0.22.2`.

**Comment by JoshClose** [comment]
I found that there is a `GetTargetFrameworks()` method on `project` that I can use. I tried that out and am getting an error now:

```
The imported project "C:\Program Files\dotnet\sdk\3.0.100\15.0\Microsoft.Common.props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.  C:\Program Files\dotnet\sdk\3.0.100\Sdks\Microsoft.NET.Sdk\Sdk\Sdk.props
```

**Comment by JoshClose** [comment]
I was able to get this to work by adding nuget package `Microsoft.Build` version `16.3.0`.

**Comment by matkoch** [comment]
That's probably #295, which will be fixed in 0.23.0.

Please consider giving the project a star :)
